libraries/AP_UAVCAN/dsdl/ardupilot/equipment/power/20004.BatteryInfoAux.uavcan

```diff
@@ -0,0 +1,35 @@
+#
+# Auxilary Single battery info.
+#
+# This message is to be used in conjunction with uavcan/equipment/power/1092.BatteryInfo.uavcan
+
+# Please ensure that this message is sent before the BatteryInfo which doesn't
+# include timestamp, following field should contain timestamp of last current measurement
+uavcan.Timestamp timestamp
+
+# Battery individual cell voltages
+# length of following field also used as cell count
+float16[<=255] voltage_cell          # [Volt]
+
+# Number of Charge Discharge Cycle
+# A charge cycle should happen when system uses all of the battery’s power.
```

Comment:
There is a problem with the text encoding in the comment, please change "‘" to "'".